fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/GithubTeamsUserRolesProvider.java

```diff
@@ -151,14 +160,33 @@ private boolean isMemberOfOrg(String username) {
   }
 
   private boolean isMemberOfTeam(Team t, String username) {
+    Integer CACHE_EXPIRES_AFTER_MINUTES = 60;
```

Comment:
I'm not opposed to the defaults, but they should be configurable. Look at the GitHubProperties object for an example (it's injected into this class as a dependency already, which is nice).

Reply:
the `ACTIVE` one should be moved to a private static member in the class, too.